taskvine/src/manager/vine_manager.c

```diff
@@ -3921,7 +3921,6 @@ struct vine_manager *vine_ssl_create(int port, const char *key, const char *cert
 	cctools_uuid_t local_uuid;
 	cctools_uuid_create(&local_uuid);
 	q->uuid = strdup(local_uuid.str);
-	q->duplicated_libraries = 0;
```

Comment:
Let's keep this counter. I'd suggest finding another way to add the taskid to the log. I don't think we want a log per task, as we would end with thousands of files.